tools/READMEs/make_readme.py

```diff
@@ -0,0 +1,40 @@
+from jinja2 import Environment, FileSystemLoader
+import json
+import os
+import sys
+
+if len(sys.argv) <= 1:
```

Comment:
Use argparse

Reply:
Done in https://github.com/YunoHost/apps/pull/1136/commits/f8c150a217aa2d81134c97011d66d55c8f53fa06